src/main/java/org/broadinstitute/hellbender/tools/sv/cluster/CanonicalSVCollapser.java

```diff
@@ -193,6 +197,21 @@ protected List<Allele> collapseAlleles(final List<Allele> altAlleles, final Alle
         return alleles;
     }
 
+    protected Set<String> collapseFilters(final List<SVCallRecord> items) {
+        return items.stream()
+                .map(SVCallRecord::getFilters)
+                .flatMap(Collection::stream)
+                .collect(Collectors.toSet());
+    }
+
+    protected Double collapseQuality(final List<SVCallRecord> items) {
+        if (items.size() == 1) {
+            return items.get(0).getLog10PError();
+        } else {
+            return null;
```

Comment:
I was conservative here by marking them as null. You could take the min, but I'd argue this could be misleading and you'd really want to go back and recalculate variant qualities after collapsing.